File: src/main/java/net/slipp/social/security/SignUpForm.java
package net.slipp.social.security;

/**
 * @author javajigi
 * 
 */
public class SignUpForm {
    private String userId;

    public SignUpForm() {
    }

    public SignUpForm(String userId) {
        this.userId = userId;
    }


    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    @Override
    public String toString() {
        return "SignUpForm [userId=" + userId + "]";
    }
}

File: src/main/scala/net/slipp/social/security/SignUpForm.scala
package net.slipp.social.security

import scala.beans.BeanProperty

class SignUpForm(uId: String) {
  @BeanProperty
  var userId = uId

  override def toString: String = {
    return "SignUpForm [userId=" + userId + "]"
  }

  def this() = this(null)
}
